styles/components/statistic.module.scss

```diff
@@ -0,0 +1,44 @@
+@use "/styles/colors";
+
+.article {
+	background-color: colors.$secondary;
+	width: 17%;
+	height: fit-content;
+	aspect-ratio: 1/1;
+	border-radius: 1vw;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	padding: 1%;
+	box-sizing: border-box;
+
+	p {
+		&:nth-child(1) {
+			font-size: 6.5vw;
```

Comment:
The stats shouldn't shrink in size on Desktop mode. They should just wrap if they overflow

Reply:
noted